ui/src/components/models/ModelVersionStatusDisplay.tsx

```diff
@@ -10,27 +10,30 @@ import React from 'react';
 export default function ModelVersionStatusDisplay({
   status,
 }: {
-  status: string;
+  status: ModelVersionStatus;
 }) {
   switch (status) {
-    case ModelVersionStatus.READY:
+    case 'READY':
```

Comment:
Is there a way to keep using the `ModelVersionStatus` enum for these cases instead of the strings?

Reply:
I generally prefer using `union` over `enum`, but considering the unusual constant values (such as `TokenType`, e.g., `urn:ietf:params:oauth:token-type:access_token`), you're absolutely right — `enum` would be more appropriate in this case. Thank you for the suggestion! Of course, we can output it as an `enum`.